README.md

```diff
@@ -89,6 +89,32 @@ Available parameters:
 * `s3_region_name: String` - provide the region for AWS S3 bucket
 * `start_block_height: u64` - block height to start the stream from
 
+## Cost estimates
+
+**TL;DR** approximately $18.15 per month (for AWS S3 access, payed directly to AWS)
```

Comment:
```suggestion
**TL;DR** approximately $18.15 per month (for AWS S3 access, paid directly to AWS) for the reading of fresh blocks
```

Reply:
Major: it's not true since we will make `list` request more frequently :(

Reply:
> I think it's a good idea also to provide the cost of reading the data from genesis till some date (we should take into account that 4 shards were introduced just recently)

IMO the most important and most frequent use case is to run an indexer following the tip of the network. So I think if anyone except us needs to read from genesis they can follow the logic of calculations to estimate the cost for their usecase.